centralized_settings.json

```diff
@@ -0,0 +1,24 @@
+{
+	"pause_all": false,
+	"pause_periods": [
+	  {
+		"name": "woo_update_6_1",
+		"start": "2024-02-16 00:00:00",
+		"end": "2024-02-20 23:59:59"
+	  }
+	],
+	"pause_times": {
+	  "start": "10",
+	  "end": "23",
+	  "friday_end": "19"
```

Comment:
I'm not sure that this is generic enough ... many countries have different weekend rules and thus a `friday_end` property doesn't always apply even though it fits our team's workflow: https://en.wikipedia.org/wiki/Workweek_and_weekend

Even the assumption that the weekend is adjacent days doesn't always hold: `[...] Friday and Sunday (in [Brunei Darussalam](https://en.wikipedia.org/wiki/Brunei), [Aceh](https://en.wikipedia.org/wiki/Aceh) province (Indonesia) and state of [Sarawak](https://en.wikipedia.org/wiki/Sarawak) (Malaysia))`

Also, I think that the name is confusing. At first, I think it meant times during which **no updates** took place but it means the exact opposite after inspecting the code, namely that updates happen only during this time.

Reply:
I don't think accounting for how other countries to weekends is an important use of energy at this point, since probably only our team is using this at this time.

However, I agree the "pause times" is the wrong wording. It's backwards! It should probably be "update times."